Client/mods/deathmatch/StdInc.h

```diff
@@ -50,6 +50,9 @@
 #include "CIdArray.h"
 #include "pcrecpp.h"
 
+// fbx
+#include "fbx/ofbx.h"

```

Comment:
I think we want to avoid adding new header files to StdInc.h wherever possible.

the others in this file are fine, since we haven't established a new convention for luadefs or CClientBlah, but you should try to remove this line specifically from this file